src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4106,7 +4106,167 @@
     <value>Unable to calculate the range for this function.</value>
     <comment>Error displayed when Range is not returned from the analyzer.</comment>
   </data>
-  <data name="equationAnalysisBack.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+  <data name="Overflow" xml:space="preserve">
+    <value>Overflow, the number is too large.</value>
+    <comment>Error that occurs during graping when the number is too large. To see this error, assign a large number to variable a, then keep doing "a:=a*a" until it happens.</comment>
+  </data>
+  <data name="RequireRadiansMode" xml:space="preserve">
+    <value>Radians mode is required to graph this equation.</value>
+    <comment>Error that occurs during graping when radians is required.</comment>
+  </data>
+  <data name="TooComplexToSolve" xml:space="preserve">
+    <value>The equation is too complex to graph.</value>
```

Comment:
I think we should use function instead of equation. Here and below.

Reply:
Also, I think we may want to say "This function..." here and below.